spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -374,7 +375,7 @@ public String createCountQueryFor(@Nullable String countProjection) {
 		/*
 		  We only support count queries for {@link PlainSelect}.
 		 */
-		if (!(selectStatement.getSelectBody()instanceof PlainSelect selectBody)) {
+		if (!(selectStatement.getSelectBody() instanceof PlainSelect selectBody)) {
 			return this.query.getQueryString();
```

Comment:
Whereas I sympathize with the desire to introduce this space WHICH SHOULD BE HERE, our current tools, due to `getSelectBody()` (the parentheses) will take that space back out the next time I'm inside this class and I reformat it. Hence, I'd rather not introduce that churn.